installer/debian/control

```diff
@@ -1,4 +1,4 @@
-Package: algorand
+Package: @PKG_NAME@
```

Comment:
this seems confusing, why is PKG_NAME a standin for 'stable' in 51algorand-updates but replacing 'algorand' here? Is it that 'stable' should have been 'algorand-stable' ?

Reply:
Yeah, that's not right, I need to fix this.